swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    def get_by_category(self, category):
+        # alternative solution: 
+        # list_of_items = []
+        # for item in self.inventory:
+        #     if item.category == category:
+        #         list_of_items.append(item)
+        # return list_of_items
+
+        list_of_items = [item for item in self.inventory 
```

Comment:
👍 Thank you for showing me the alternative, so that I know you understand what this list comprehension is doing!